test-iss.sh

```diff
@@ -1,45 +1,71 @@
-
 #!/bin/bash
 
-cd test
-make clean > /dev/null 2>&1
-cd ..
+  #
+  # Helper functions
+  #
 
-TESTS="test/2d-bidirec.c \
-test/reverse-iss.c
-"
+  cleanup() {
+    rm -f test_temp_out.pluto.c
+    rm -f test_temp_out.pluto.pluto.cloog
+  }
 
-OPTS="--iss --islsolve"
+  # Runs command cmd and updates global exit_value on error
+  run_test() {
+    local file=$1
+    local cmd=$2
+    local test_exit_value
 
-for file in $TESTS; do
-	echo -e "$file" 
-    echo ./polycc $OPTS $file $*  -o test_temp_out.pluto.c
-    ./polycc $OPTS $file $*  -o test_temp_out.pluto.c
-    if [ $? -ne 0 ]; then
-        echo Failed test case "$file"!
-        break
-    fi
-done
-
-OPTS="--iss --islsolve --lbtile --parallel"
-for file in $TESTS; do
-	echo -e "$file" 
-    echo ./polycc $OPTS $file $*  -o test_temp_out.pluto.c
-    ./polycc $OPTS $file $*  -o test_temp_out.pluto.c
-    if [ $? -ne 0 ]; then
-        echo Failed test case "$file"!
-        break
+    # Log information
+    echo "$file"
+    echo "$cmd"
+
+    # Execute test
+    $cmd
+
+    # Check exit value
+    test_exit_value=$?
+    if [ ${test_exit_value} -ne 0 ]; then
+      echo "Failed test case $file!"
+      exit_value=${test_exit_value}
```

Comment:
Whitespace changes throughout.

Reply:
Removed trailing whitespaces in `test-iss-sh` and `test.sh`